zpages/rpcz.go

```diff
@@ -258,39 +260,55 @@ func (s snapExporter) ExportView(vd *view.Data) {
 
 		// Update field of s corresponding to the view.
 		switch vd.View {
-		case ocgrpc.ClientErrorCountView:
-			s.ErrorsTotal = int(count)
+		case ocgrpc.ClientCompletedRPCsView:
+			if _, ok := haveResetErrors[method]; ok {
+				haveResetErrors[method] = struct{}{}
+				s.ErrorsTotal = 0
+			}
+			for _, tag := range row.Tags {
+				if tag.Key == ocgrpc.KeyClientStatus && tag.Value != "OK" {
+					s.ErrorsTotal += int(count)
```

Comment:
Although we removed the measure and view for viewing the total number of gRPC errors.

Reply:
Yes, so this is why I had to compute it from the CompletedRPCs measures

Reply:
I have a naive question: how are we then going to view and track the number of errors?